install.sh

```diff
@@ -392,10 +392,11 @@ if [ "$1" = "install" ]; then
         fi
         _headers_rpm="kernel-headers-${_kernelname}*.rpm"
         _kernel_rpm="kernel-${_kernelname}*.rpm"
+        _kernel_devel_rpm="kernel-${_kernelname}*.rpm"
```

Comment:
Looks like a copy-pasta error here :yum:

Reply:
oh whoopsie. your PR looks better overall anyway ;)

Reply:
Suse would have had the same issue so it needed to be handled too, thanks for reporting!